packages/core/contracts/optimistic-asserter/interfaces/OptimisticAsserterInterface.sol

```diff
@@ -151,7 +151,8 @@ interface OptimisticAsserterInterface {
         address caller,
         uint64 expirationTime,
         IERC20 currency,
-        uint256 bond
+        uint256 bond,
+        bytes32 identifier
```

Comment:
It might be quite useful to have this as indexed parameter for bots to be able filtering. Would it make sense dropping index from escalationManager and add it to the identifier?

Reply:
I think you are right, indexing `identifier` sounds more useful